### PR TITLE
Add snippets for datasource and Jaeger properties configuration

### LINKED ISSUE #90: Add snippet for configuring Panache
Create a custom snippet to set the configuration properties for Panache:

```
"Add Panache properties": {
		"prefix": "qpm",
		"body": [
			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
			"quarkus.datasource.driver=${2:org.mariadb.jdbc.Driver}",
			"quarkus.datasource.username=${3:developer}",
			"quarkus.datasource.password=${4:developer}",
			"quarkus.hibernate-orm.database.generation=${4:update}"
		],
		"description": "Configure Quarkus Panache"
	}
```

### LINKED ISSUE #93: Add snippet for Jaeger properties configuration
Add configuration properties for Jaeger.

```
"Add Jaeger properties": {
		"prefix": "qj",
		"body": [
			"quarkus.jaeger.service-name=${1:myservice}",
			"quarkus.jaeger.sampler-type=${2:const}",
			"quarkus.jaeger.sampler-param=${3:1}",
			"quarkus.jaeger.endpoint=${4:http://localhost:14268/api/traces}"
		],
		"description": "Configure Quarkus Jaeger"
	}
```

### PR DESCRIPTION
**Pull request by fbricon**
Fixes #90 (Datasource snippet)
Fixes #93 (Jaeger snippet)

### CODE REVIEW

**Comment by fbricon** [comment]
@lordofthejars since configuring datasource properties is not Panache-specific, I changed the description to a more generic "Configure Quarkus datasource"
Screen Shot 2019-09-25 at 10 47 06 AM

File: snippets/properties.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
+			"quarkus.datasource.driver=${2:org.mariadb.jdbc.Driver}",
```
Comment (angelozerr) [inline snippets/properties.json]: Why not using choice https://code.visualstudio.com/docs/editor/userdefinedsnippets#_choice to provide a list of JDBC driver?

File: snippets/properties.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
```
Comment (angelozerr) [inline snippets/properties.json]: Why not using choice https://code.visualstudio.com/docs/editor/userdefinedsnippets#_choice to provide a list of syntax URL (according JDBC driver)?

File: snippets/properties.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+			"quarkus.datasource.driver=${2:org.mariadb.jdbc.Driver}",
+			"quarkus.datasource.username=${3:developer}",
+			"quarkus.datasource.password=${4:developer}",
+			"quarkus.hibernate-orm.database.generation=${5:update}"
```
Comment (angelozerr) [inline snippets/properties.json]: Use choice to provide the list `none, create, drop-and-create, drop, update`

Comment (fbricon) [reply snippets/properties.json]: excellent idea, this works well

**Comment by fbricon** [comment]
now with choices
ds-snippet

**Comment by fbricon** [comment]
included commit for Jaeger snippet

File: snippets/properties.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1|jdbc:mariadb://localhost:3306/mydb,jdbc:mysql://localhost:3306/test,jdbc:h2:mem:mydb,jdbc:postgresql://localhost/mydb,jdbc:sqlserver://localhost:1433;databaseName=mydb|}",
+			"quarkus.datasource.driver=${2|org.mariadb.jdbc.Driver,org.mysql.jdbc.Driver,org.h2.Driver,org.postgresql.Driver,com.microsoft.sqlserver.jdbc.SQLServerDriver|}",
```
Comment (angelozerr) [inline snippets/properties.json:6]: that's great it works! What about other JDBC driver like derby? Perhaps we will add another JDB driver according user feedback?

Comment (lordofthejars) [reply snippets/properties.json:6]: I think Derby is not supported yet in Quarkus.

Comment (angelozerr) [reply snippets/properties.json:6]: Thanks @lordofthejars for your feedback! We will add another database according user feedback.

Comment (fbricon) [reply snippets/properties.json:6]: I listed the drivers according to the quarkus extensions found in https://quarkus.io/guides/datasource-guide#jdbc-url-configurations

**Pull request merged**